### PR TITLE
add scatterplot

### PR DESCRIPTION
**Pull request by zindlerb**

### CODE REVIEW

**Comment by zindlerb** [comment]
:new_moon_with_face: :fire: :full_moon_with_face:

File: client/js/components/data_page.jsx [hunk 1]
```diff
@@ -109,6 +109,7 @@ var Table = React.createClass({
   },
   componentDidMount: function() {
     AppStore.renderSchema(this.props.schemaName);
+    //Refactor THIS!!
```
Comment (dduugg) [inline client/js/components/data_page.jsx]: Make this a TODO and include the issue #

File: client/js/components/chart.jsx [hunk 2]
```diff
@@ -80,6 +81,50 @@ var Chart = React.createClass({
...
+          xArray.push(row[xIndex]);
+          yArray.push(row[yIndex]);
+        });
+
+        return [xArray, yArray]
+      }
+      
+      var tplot = transformCSVtoScatterPlot(csvData, schemaName, xIndex, yIndex);
+      var xs = {};
+      xs[schemaName] = schemaName + "_x";
```
Comment (dduugg) [inline client/js/components/chart.jsx]: are you as excited for computed property types in ES6 as i am?

File: client/js/components/chart.jsx [hunk 3]
```diff
@@ -80,6 +81,50 @@ var Chart = React.createClass({
...
+
+        var xArray = [schemaName + "_x"];
+        var yArray = [schemaName];
+
+        _.each(nonHeaderData, function(row) {
+          xArray.push(row[xIndex]);
+          yArray.push(row[yIndex]);
+        });
+
+        return [xArray, yArray]
```
Comment (dduugg) [inline client/js/components/chart.jsx]: missing `;`

**Comment by dduugg** [comment]
:+1:

**Comment by cmperez** [comment]
feel free to bang bang after dug's changes :+1:

File: client/js/components/chart.jsx (hunk 2 above)
Comment (zindlerb) [inline client/js/components/chart.jsx]: :heart_eyes:

**Comment by zindlerb** [comment]
Ch-ch-changes made!

**Pull request merged**